core/tests/karma.conf.js

```diff
@@ -24,7 +24,7 @@ module.exports = function(config) {
       'third_party/static/angularjs-1.5.8/angular-mocks.js',
       'third_party/static/headroom-js-0.9.4/headroom.min.js',
       'third_party/static/headroom-js-0.9.4/angular.headroom.min.js',
-      'third_party/static/math-expressions-370a77/build/math-expressions.js',
+      'third_party/static/math-expressions-v1.7.0/math-expressions.js',
```

Comment:
Generally we don't have a "v" in the suffix -- see the other imports, for example.

Reply:
Fixed!